docs/user-guide/creating-application/deployment-template.md

```diff
@@ -575,3 +575,52 @@ If you want to see application metrics like different HTTP status codes metrics,
 
 Once all the Deployment template configurations are done, click on `Save` to save your deployment configuration. Now you are ready to create [Workflow](workflow/) to do CI/CD.
 
+### Helm Chart Json Schema
+
+Helm Chart json schema is used to validate the deployment template values.
+
+```
```

Comment:
create table for chart version-json schema link

document any other validation which is not in the schema